src/components/icon/js/iconDirective.js

```diff
@@ -96,7 +96,7 @@ angular
  * determine its textual name and character reference code. Click on any icon to see the slide-up information
  * panel with details regarding a SVG download or information on the font-icon usage.
  *
- * <a href="https://www.google.com/design/icons/#ic_accessibility" target="_blank" style="border-bottom:none;">
+ * <a href="https://material.io/archive/guidelines/usability/accessibility.html" target="_blank" style="border-bottom:none;">
```

Comment:
This is actually about icons, not accessibility. The link should be https://material.io/tools/icons/?icon=accessibility&style=baseline

Reply:
This has been updated :chicken: